examples/plot_simulate_lfp_array.py

```diff
@@ -0,0 +1,106 @@
+"""
+==============
+08. Record LFP
+==============
+
+This example demonstrates how to record local field potentials (LFPs).
+"""
+
+# Authors: Nick Tolley <nicholas_tolley@brown.edu>
+#          Mainak Jas <mainakjas@gmail.com>
+#          Christopher Bailey <cjb@cfin.au.dk>
+
+# sphinx_gallery_thumbnail_number = 3
+
+import os.path as op
+
+###############################################################################
+# Let us import ``hnn_core``.
+
+import hnn_core
+from hnn_core import read_params, Network, simulate_dipole
+from hnn_core.parallel_backends import MPIBackend
+
+hnn_core_root = op.dirname(hnn_core.__file__)
+
+###############################################################################
+# Then we read the parameters file
+# params_fname = op.join(hnn_core_root, 'param', 'default.json')
+params_fname = op.join(hnn_core_root, 'param', 'gamma_L5weak_L2weak.json')
+params = read_params(params_fname)
+params['tstop'] = 250
+###############################################################################
+# We will start with simulating the evoked response from
+# :ref:`evoked example <sphx_glr_auto_examples_plot_simulate_evoked.py>`.
+# We first instantiate the network.
+# (Note: Setting ``add_drives_from_params=True`` loads a set of predefined
+# drives without the drives API shown previously).
+net = Network(params)
+weights_ampa = {'L2_pyramidal': 0.0008, 'L5_pyramidal': 0.0075}
+synaptic_delays = {'L2_pyramidal': 0.1, 'L5_pyramidal': 1.0}
+rate_constant = {'L2_pyramidal': 140.0, 'L5_pyramidal': 40.0}
+net.add_poisson_drive(
+    'poisson', rate_constant=rate_constant, weights_ampa=weights_ampa,
+    location='proximal', synaptic_delays=synaptic_delays, seedcore=1079)
+
+###############################################################################
+# LFP recordings require specifying the electrode postion. It can be useful
+# to visualize the cells of the network to decide on the placement of each
+# electrode.
+net.plot_cells()
+
+###############################################################################
+# Electrode positions are stored under ``Network.lfp_array`` as a list
+# of tuples. Once we have chosen x,y,z coordinates for each electrode, we can
+# add them to the simulation.
+
+# XXX coordinates FUBAR, cortical depth direction is Y in NEURON objects!
+# electrode_pos = [(2, 400, 2), (6, 800, 6)]
+depths = list(range(-400, 1900, 200))
+electrode_pos = [(45, dep, 45) for dep in depths]
+net.add_electrode_array('arr1', electrode_pos)
+print(net.lfp_array)
+net.plot_cells()
+
+with MPIBackend(n_procs=2):
+    dpl = simulate_dipole(net)
+###############################################################################
+# Now that our electrodes are specified, we can run the simulation. The LFP
+# recordings are stored under ``Network.lfp_array``.
```

Comment:
I wouldn't show users two different ways of plotting the data. Just showing our `plot` method is sufficient I would say